config/initializers/publish_pre_production_finders_feature_flag.rb

```diff
@@ -0,0 +1 @@
+SpecialistPublisher::Application.config.publish_pre_production_finders = ENV["PUBLISH_PRE_PRODUCTION_FINDERS"].present?
```

Comment:
I like this. I'm going to steal this idea of moving all `ENV['VAR']` usage to an initializer :+1: